lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -21,10 +21,21 @@ class BaseI18nTestCase(TestCase):
     """
     Base utilities for i18n test classes to derive from
     """
+    def __init__(self, *args, **kwargs):
+        super(BaseI18nTestCase, self).__init__(*args, **kwargs)
+        self.user = User()
```

Comment:
Can remove this line to create the user since it is created below using `UserFactory`.